crates/gitbutler-tauri/src/projects.rs

```diff
@@ -96,6 +97,19 @@ pub mod commands {
     pub fn delete_project(projects: State<'_, Controller>, id: ProjectId) -> Result<(), Error> {
         projects.delete(id).map_err(Into::into)
     }
+
+    #[tauri::command(async)]
+    #[instrument(skip(projects))]
+    pub fn get_pr_template_contents(
+        projects: State<'_, Controller>,
+        relative_path: &path::Path,
+        project_id: ProjectId,
+    ) -> Result<String, Error> {
+        let project = projects.get(project_id)?;
+        let template_path = project.path.join(relative_path);
+
+        Ok(read_file_from_workspace(template_path.as_path())?)
```

Comment:
Talking to Nico, we've acknowledged that a path could back out of the directory using `../../../`, but due to the size of the PR, we'd rather merge in now, and improve the security in a follow up PR.